plugins/ibm_qradar/requirements.txt

```diff
@@ -0,0 +1,5 @@
+# List third-party dependencies here, separated by newlines. 
+# All dependencies must be version-pinned, eg. requests==1.2.0
+# See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
+
+insightconnect-plugin-runtime
```

Comment:
The plugin runtime does not need to be included in the requirements.txt because the Docker environment in which plugins run always adds this package by default.

```suggestion
```